.github/workflows/ruby_on_rails.yml

```diff
@@ -22,7 +22,7 @@ jobs:
         ruby-version: ${{ matrix.ruby_version }}
     - name: Build and test with Rake
       run: |
-        gem install bundler:1.14.0
+        gem install bundler:1.17.3
```

Comment:
@joelhawksley I hope this change is OK, I noticed it was changed in #35 so maybe it was a ruby 2.3 support thing?

But now it matches what's in `Gemfile.lock`

Reply:
That's fine. Might you break this into its own PR?

Reply:
👍 no prob!

Reply:
I extracted this change in to #200 I'll rebase when that's merged.

Reply:
OK, I rebased! :)